### PR TITLE
Add Itinerary flag to enable error on accessible trips

### PR DESCRIPTION
**Pull request by miles-grant-ibigroup**
**Description:**

Accessible trips can be funky, and this PR allows us to create a new error whenever we get accessible results. This can be used to warn users of poor accessible routing quality.

**PR Checklist:**
- [ ] Does the code follow accessibility standards (WCAG 2.1 AA Compliant)?
- [ ] Are all languages supported (Internationalization/Localization)?
- [ ] Are appropriate Typescript types implemented?

### CODE REVIEW

File: example-config.yml [hunk 1]
```diff
@@ -362,6 +362,8 @@ itinerary:
   showScheduleDeviation: true
   # Shows the duration of a leg below the leg in the metro itinerary summary
   showLegDurations: false
+  # Whether to add a OTP_RR_A11Y_ROUTING_ENABLED error to all itineraries with accessibility scores
```
Comment (amy-corson-ibigroup) [inline example-config.yml:365]: Can we add a note here that `OTP_RR_A11Y_ROUTING_ENABLED` will also need to be added in config?

**Review by amy-corson-ibigroup — APPROVED** [review]
Looks good!

File: example-config.yml (hunk 1 above)
Comment (miles-grant-ibigroup) [reply example-config.yml:365]: I think it's clear if it's missing (big error in the html()

File: lib/components/narrative/metro/metro-error-renderer.tsx
Comment (amy-corson-ibigroup) [inline lib/components/narrative/metro/metro-error-renderer.tsx:1]: Nitpicky but is it possible to bold this link? it's a little difficult at first glance to tell what's going on in the error

Comment (miles-grant-ibigroup) [reply lib/components/narrative/metro/metro-error-renderer.tsx:1]: It's possible, but this is a generic feature so we can't bold things in every case

Comment (amy-corson-ibigroup) [reply lib/components/narrative/metro/metro-error-renderer.tsx:1]: Sure, but probably in every case if you're adding a link you're going to want that link to stand out from the rest of the error

File: lib/actions/apiV2.js [hunk 2]
```diff
@@ -1011,6 +1011,24 @@ export function routingQuery(searchId = null, updateSearchInReducer) {
                   response.data?.plan?.routingErrors.filter(
                     (re) => re?.code !== 'NO_TRANSIT_CONNECTION'
                   )
+
+                // Add accessibility error, if it is turned on
+                const state = getState()
+                const { displayA11yError } = state.otp.config?.itinerary
+                if (displayA11yError) {
+                  if (
```
Comment (AdrianaCeric) [inline lib/actions/apiV2.js:1019]: this is a lot of if statements maybe use `&&`?

**Review by AdrianaCeric — APPROVED** [review]
👍

File: lib/actions/apiV2.js (hunk 2 above)
Comment (miles-grant-ibigroup) [reply lib/actions/apiV2.js:1019]: I think it's cleaner like this!

**Pull request merged**